pickfirst.go

```diff
@@ -131,6 +133,17 @@ func (p *picker) Pick(info balancer.PickInfo) (balancer.PickResult, error) {
 	return p.result, p.err
 }
 
+// idlePicker is used when the SubConn is IDLE and kicks the SubConn into
+// CONNECTING when Pick is called.
+type idlePicker struct {
+	sc balancer.SubConn
+}
```

Comment:
Would every balancer implementation need to do something like this to kick the clientconn out of idle?

Reply:
As discussed offline, different balancers will handle idle transitions differently.  Some will want something like this.  Others, like RR, will continuously reconnect.